index.js

```diff
@@ -1,5 +1,5 @@
 function setState(store, newState, afterUpdateCallback) {
-  store.state = { ...store.state, ...newState };
+  store.state = Object.assign(store.state, newState);
```

Comment:
@andregardi  and. just FYI: one to one fix would have been Object.assign({}, store.state, newState); but I decided to mutate state partially. Hit me if you see problem there.